test/functional/helpers.lua

```diff
@@ -45,6 +45,8 @@ module.nvim_argv = {
   '--cmd', 'unmap <C-L>',
   '--cmd', 'iunmap <C-U>',
   '--cmd', 'iunmap <C-W>',
+  '--cmd', 'xunmap *',
+  '--cmd', 'xunmap #',
```

Comment:
This is already running out of `--cmd` in a test, which is why all CI checks are failing.